Fix for parameter hints not showing when using tabs instead of spaces

SietseB opened the pull request:
Parameter hints are not showing when using the tab character for indents.
In case of spaces a column number equals the character position in the line string, but in case of tabs that trick doesn't work.
This fix converts a column number to a character position, taking tabs into account.

SietseB committed:
Fix for parameter hints not showing when using tabs instead of spaces

Parameter hints were not showing when using the tab character for indents.
In case of spaces a column number equals the character position in the line string, but in case of tabs that trick doesn't work.
This fix converts a column number to a character position, taking tabs into account.

swmitra reviewed (approved):
Interesting solution to handle tabs 👍   
Change looks good to me, please test JS intellisense features once  before merging.

SietseB committed:
Code clean-up: using Editor method to convert column number to char pos

Using the build-in method Editor.getCharIndexForColumn results in much shorter and cleaner code.

yogo1212 commented:
@SietseB i've squashed and merged your changes into https://github.com/brackets-cont/brackets
